build.sbt

```diff
@@ -159,7 +165,7 @@ lazy val ratatoolCommon = project
       "org.apache.avro" % "avro" % avroVersion,
       "org.apache.avro" % "avro-mapred" % avroVersion classifier "hadoop2",
       "com.google.guava" % "guava" % guavaVersion,
-      "com.google.apis" % "google-api-services-bigquery" % bigqueryVersion % Test,
```

Comment:
Why this change? It does expand the scope of potential dependency conflicts

Reply:
although I could refactor the `getFieldModeWithDefault` to just accept a mode string and perform the logic on the passed in mode, which would avoid the need to make the bigquery API available to the ratatoolCommon module

Reply:
should be resolved now